KarhooUISDK/Screens/BookingScreen/QuoteListMVP/KarhooQuoteListViewController.swift

```diff
@@ -121,7 +121,7 @@ final class KarhooQuoteListViewController: UIViewController, QuoteListView {
             _ = stackConstraints.map { $0.priority = .defaultLow }
             _ = stackConstraints.map { $0.isActive = true }
             
-            _ = [legalDisclaimerLabel.heightAnchor.constraint(equalToConstant: 55.0)].map { $0.isActive = true }
+            _ = [legalDisclaimerLabel.heightAnchor.constraint(equalToConstant: 30.0)].map { $0.isActive = true }
```

Comment:
I think this would be a good time to use `UIConstants`. Since this is a new value you can add a let for `smallRowHeight: CGFloat = 30` under `Dimensions.View` and use that from now on for small height rows.